include/pm/device_runtime.h

```diff
@@ -42,7 +42,6 @@ void pm_device_runtime_enable(const struct device *dev);
  *
  * @retval 0 If the device runtime PM is disabled successfully.
  * @retval -ENOSYS If the functionality is not available.
- * @retval -EPERM If the device pm state is locked.
```

Comment:
since change is added, then removed, can initial commit be fixed up so that the only the final behavior is introduced in this PR? It's now a bit confusing.

Reply:
ok, will sqaush them,.